internal/pproflogging/pproflogging.go

```diff
@@ -63,23 +63,29 @@ const (
 	ProfileNameCPU = "cpu"
 )
 
-// ProfileConfig configuration flags for a profile.
-type ProfileConfig struct {
-	flags []string
-	buf   *bytes.Buffer
+var (
+	// ErrEmptyProfileName returned when a profile configuration flag has no argument.
+	ErrEmptyProfileName = errors.New("empty profile flag")
+
+	//nolint:gochecknoglobals
+	pprofConfigs = newProfileConfigs(os.Stderr)
+)
+
+// Writer interface supports destination for PEM output.
+type Writer interface {
+	io.Writer
+	io.StringWriter
 }
 
 // ProfileConfigs configuration flags for all requested profiles.
 type ProfileConfigs struct {
 	mu sync.Mutex
-
+	// +checklocks:mu
+	wrt Writer
```

Comment:
Could you please add a comment right above `wrt` to help the readers understand what is it about ?

Reply:
I can add that in a followon PR with some additional unit testing and comments.

Reply:
@aaron-kasten I'd prefer that we either add the comment here or you can file an issue in Kopia and reference it here to track it, so we don't miss this. Otherwise, it'd be hard to track this comment.

Marking unresolved.